tests/test_process_new_granules.py

```diff
@@ -128,6 +128,7 @@ def test_get_jobs_for_granule():
     payload = process_new_granules.get_jobs_for_granule(subscription, granule)
     assert payload == [
         {
+            'subscription_id': 'f00b731f-121d-44dc-abfa-c24afd8ad542',
```

Comment:
Slightly pedantic, but I think it would be worth using different `subscription_id` values for the rtc and insar test cases to confirm that the appropriate id gets attached to the corresponding jobs